src/main/groovy/edu/oregonstate/mist/osuevents/resources/EventsResource.groovy

```diff
@@ -66,14 +74,116 @@ class EventsResource extends Resource {
  * GET all events
  */
     @GET
-    @Produces(MediaType.APPLICATION_JSON)
+    @Produces([MediaType.APPLICATION_JSON, "text/csv"])
+    //TODO Add CSV MediaType to response object
     public Response getEvents(@Auth AuthenticatedUser _,
                               @QueryParam('format') String format) {
-        def events = eventsDAO.getEvents()
-//@TODO add methods for returning csv and ics formats
-//        else if (format == "csv") {
-//
-//        } else if (format == "ics") {
+        List<ResourceObject> events = eventsDAO.getEvents()
+        def resultObject
+
+        if (!format) {
+            events.each {
+                resultObject.data += new ResourceObject(
+                        id: it.eventID,
+                        type: 'event',
+                        attributes: it
+                )
+            }
+        } else if (format == "csv") {
+            final ArrayList<String> BASECSVHEADER =
```

Comment:
We aren't using "Allow User Attendance", "Allow User Activity", or "Featured Tabs", we should remove those from the csv header and remove the placeholder values being populated for each event.